system/Model.php

```diff
@@ -714,20 +714,17 @@ public function __isset(string $name): bool
      */
     public function __call(string $name, array $params)
     {
-        $result = parent::__call($name, $params);
+        $builder = $this->builder();
+        $result  = null;
 
-        if ($result === null && method_exists($builder = $this->builder(), $name)) {
+        if (method_exists($this->db, $name)) {
+            $result = $this->db->{$name}(...$params);
+        } else if (method_exists($builder, $name)) {
             $result = $builder->{$name}(...$params);
-        }
-
-        if (empty($result)) {
-            if (! method_exists($this->builder(), $name)) {
-                $className = static::class;
-
-                throw new BadMethodCallException('Call to undefined method ' . $className . '::' . $name);
-            }
+        } else {
+            $className = static::class;
 
```

Comment:
```suggestion
```

Reply:
@paulbalandan 

Thanks, review.

Fixed it.